lib/matplotlib/axes/_base.py

```diff
@@ -1499,7 +1499,7 @@ def set_facecolor(self, color):
 
         Parameters
         ----------
-        color : color
+        color : color defined by string name, char abbreviation or hex, see :doc:`gallery/color/named_colors`
```

Comment:
So we have _many_ of these, where `color : color`.  

I think there is an argument for adding links: 

```
color : color; see :ref:`colors_def`
```
would be a better link (resolves to https://matplotlib.org/stable/users/explain/colors/colors.html).  

I know this was discussed before, but I forget the resolution.  @timhoffm is the API documentation lead.

Reply:
See https://github.com/matplotlib/matplotlib/issues/24859#issuecomment-1368579653

Reply:
Right so I think the suggestion above is fine. 

Bonus points if this PR searches and replaces the other instances

Reply:
I can do it tomorrow...

Reply:
btw, there are also many cases like
```
color : color
	Color for axis pane.
```
shall this be also extended?
```
color : color; see :ref:`colors_def`
	Color for axis pane.
```
or
```
color : color
	Color for axis pane. See :ref:`colors_def`.
```